docker-compose.prod.yml

```
@@ -0,0 +1,49 @@
+  redis:
+    restart: always
+    command: redis-server --appendonly yes --maxmemory 512mb --maxmemory-policy allkeys-lru --requirepass ${REDIS_PASSWORD:?Missing REDIS_PASSWORD}
+
+  # Qdrant - Production settings
+  qdrant:
+    restart: always
+    environment:
+      QDRANT__SERVICE__HTTP_PORT: 6333
+      QDRANT__SERVICE__GRPC_PORT: 6334
+      QDRANT__LOG_LEVEL: WARN
+      QDRANT__SERVICE__API_KEY: ${QDRANT_API_KEY:?Missing QDRANT_API_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)